lib/modules/manager/pixi/extract.ts

```diff
@@ -0,0 +1,69 @@
+import { logger } from '../../../logger';
+import { getSiblingFileName, localPathExists } from '../../../util/fs';
+import { Result } from '../../../util/result';
+import { Toml } from '../../../util/schema-utils';
+import { PyProjectSchema } from '../pep621/schema';
+import type { PackageFileContent } from '../types';
+import { type PixiConfig, PixiToml } from './schema';
+
+const PyProjectToml = Toml.pipe(PyProjectSchema);
+
+function getUserPixiConfig(
+  content: string,
+  packageFile: string,
+): null | PixiConfig {
+  if (packageFile.endsWith('pyproject.toml')) {
```

Comment:
```suggestion
  if (packageFile === 'pyproject.toml' || packageFile.endsWith('/pyproject.toml')) {
```
make it safer to not match `some-pyproject.toml`